cql3/restrictions/statement_restrictions.cc

```diff
@@ -1394,6 +1395,94 @@ bool is_satisfied_by(
         }, restr);
 }
 
+/// A column's bound, from WHERE restrictions.
+class bound_t {
+    bool _unbounded;
+    bytes_opt _value; // Invalid when _unbounded is true.
```

Comment:
What does nullopt mean here? The user compared `column <= NULL`?

Reply:
Yes, in anticipation of that becoming allowed one day.

Reply:
bound_t went away in subsequent versions...